db/config.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+    host: 'localhost',
+    user: 'root',
+    password: 'password',
```

Comment:
Recomended to add your password in a config.js file that is also on your gitignore so your password doesnt make its way into the public realm, even if it is just a dummy password